docs/misconfiguration/iac.md

```diff
@@ -0,0 +1,157 @@
+# Infrastructure as Code (IaC)
+
+## Quick start
+
+Simply specify a directory containing IaC files such as Terraform and Dockerfile.
+
+``` bash
+$ trivy config [YOUR_IaC_DIRECTORY]
+```
+
+Then, Trivy pulls built-in policies from GitHub Container Registry as OCI artifact and scans IaC files with those policies.
```

Comment:
I think the details of how the policy db works is unnecessary for the quick start

```suggestion
Trivy will automatically fetch the latest policy database and will keep it up-to-date in future scans.
```

Reply:
done https://github.com/aquasecurity/trivy/pull/1101/commits/1c9fddb847c89159918743f11c9eec3817f32901